socorro/signature/rules.py

```diff
@@ -306,13 +303,11 @@ def _do_generate(self, source_list, hang_type, crashed_thread, delimiter=' | '):
                 debug_notes.append('irrelevant; ignoring: "{}"'.format(a_signature))
                 continue
 
-            # If the signature matches the trim dll signatures regex, rewrite it to remove all but
```

Comment:
We're switching this block to apply to all dll frame signatures rather than a specific list of them. Further, we're going to trim them all, too.